pkg/command/context.go

```diff
@@ -925,16 +925,34 @@ func deleteCtx(_ *cobra.Command, args []string) error {
 			return nil
 		}
 	}
+	ctx, err := config.GetContext(name)
+	if err != nil {
+		return err
+	}
 	installed, _, _, _ := getInstalledAndMissingContextPlugins() //nolint:dogsled
 	log.Infof("Deleting entry for context '%s'", name)
-	err := config.RemoveContext(name)
+	err = config.RemoveContext(name)
 	if err != nil {
 		return err
 	}
 	listDeactivatedPlugins(installed, name)
+	deleteKubeconfigContext(ctx)
+
 	return nil
 }
 
+func deleteKubeconfigContext(ctx *configtypes.Context) {
+	// Note: currently cleaning up the kubeconfig for TAE context types only.
+	// (Since the kubernetes context type can have kube context provided by the user, it may not be
+	// desired outcome for user if CLI deletes/cleanup kubeconfig provided by the user.)
+	if ctx.Target == configtypes.TargetTAE {
+		log.Infof("Deleting the  context '%s' from the kubeconfig file '%s'", ctx.ClusterOpts.Context, ctx.ClusterOpts.Path)
```

Comment:
I am a little uneasy that in consecutive log lines we use the term context to refer to two different things
```
[i] Deleting entry for context 'tt-ucp2'
[i] Deleting the  context 'tanzu-cli-tt-ucp2' from the kubeconfig file '/Users/pkalle/.kube/config'
```
I don't have any brilliant suggestions. But maybe the second line can be modded to
```
Deleting kubeconfig context 'tanzu-cli-tt-ucp2' from the file '/Users/pkalle/.kube/config'
```

Reply:
Updated the log as suggested. Thanks